packages/wagmi/package.json

```diff
@@ -75,7 +75,7 @@
     "@wagmi/core": ">=2.0.0",
     "react": ">=17",
     "react-dom": ">=17",
-    "viem": ">=2.0.0",
+    "viem": "2.x",
```

Comment:
Why there are two in this package.json?

Reply:
is this ok? peer dependencies should be marked as a range of versions